mbed_host_tests/host_tests/base_host_test.py

```diff
@@ -80,6 +81,19 @@ def teardown(self):
         raise NotImplementedError
 
 
+def event_cb(key):
```

Comment:
Please change from ```event_cb``` to ```event_callback```.
Niklas told me before they prefer this type of names (explicit, so no one had to quess what ```_cb``` stands for).